packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -2,18 +2,58 @@
 const expect = require('expect');
 const { accessibility } = require('../commands/validates');
 const { toBeAccessible } = require('../commands/expect');
+const viewportHelpers = require('../commands/viewport-helpers');
```

Comment:
nit: obscuring the functions onto a random helper variable here feels strange. Can this be deconstructed into the specific functions? These could probably be organized a little differently, but that's not really necessary for this PR.

```js
const { getViewports, describeViewports, setViewport } = require('../commands/viewport-helpers');
```

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/438/commits/19c190061a6d0f428d0ab4b33b669bed04d9f219